src/tests/ftest/util/ServerUtils.py

```diff
@@ -46,6 +51,79 @@ class ServerFailed(Exception):
 def printFunc(thestring):
     print("<SERVER>" + thestring)
 
+def nvme_yaml_config(default_value_set, bdev, enabled=False):
+    """
+    Enable/Disable NVMe Mode.
+    NVMe is enabled by default in yaml file.So disable it for CI runs.
```

Comment:
Not sure what the parameters are.

Reply:
I will add info about the arguments and will change the function name.